src/Form/test-cases/index.js

```diff
@@ -62,25 +62,25 @@ module.exports = [
     { html: 'espn_login.html' },
     { html: 'espn_signup.html' },
     { html: 'usps_login.html' },
-    { html: 'usps_signup.html', expectedFailures: ['unknown', 'unknown', 'unknown', 'unknown'] },
+    { html: 'usps_signup.html' },
     { html: 'usps_checkout.html' },
     { html: 'quizlet_login.html' },
     { html: 'quizlet_signup.html', expectedFailures: ['birthdayMonth', 'birthdayDay', 'birthdayYear'] },
     { html: 'quizlet_checkout.html' },
     { html: 'lowes_login.html' },
     { html: 'lowes_signup.html' },
     { html: 'lowes_checkout.html' },
-    { html: 'idme_login.html', expectedFailures: ['unknown'] },
+    { html: 'idme_login.html', expectedFailures: ['unknown', 'unknown'] },
```

Comment:
What's the reason for the new failure here?

Reply:
As I improved username detection, it detected a visually hidden username that is marked as `unknown`. As discussed, we may change these `unknown`s in the near future, so this may go away soon.